### ISSUE TITLE
Add rule: disable es6 template literals (eg ```{`foo${bar}`}```)  in attributes

### ISSUE THREAD

**Issue by mustafa0x** [sveltejs/eslint-plugin-svelte#893]
### Motivation

some developers (and ai, too!) will write attributes in jsx style. eg ```<a href={`https://example.com${path}`}>link</a>```.

in idiomatic svelte, this should be `<a href="https://example.com{path}">link</a>`

### Description

warn on ```attr={``}```

### Examples

```svelte
<script>
</script>

<!-- ✓ GOOD -->
<Foo attr="foo${bar}" />

<!-- ✗ BAD -->
<Foo attr={`foo${bar}`} />
```

**Comment by ota-meshi** [comment]
Thank you for proposing the rule!
That's sounds good to me!
